src/server/game/Scripting/ScriptMgr.cpp

```diff
@@ -308,6 +308,13 @@ UnitScript::UnitScript(const char* name, bool addToScripts)
         ScriptRegistry<UnitScript>::AddScript(this);
 }
 
+ThreatScript::ThreatScript(const char *name, bool addToScripts) : ScriptObject(name)
```

Comment:
Why not use string_view

Reply:
i dont know ? literally all of others are like that

Reply:
Ok fine for now, but I suggest that you read over the code standards, because at this stage it wont be merged due to it

Reply:
im aware of that i was asking about it on discord and in the end i decided to just try creating PR to know for sure if it will be issue or not